slang-api/src/main/java/com/sonarsource/slang/visitors/TreePrinter.java

```diff
@@ -54,6 +56,8 @@ public static String tree2string(Tree tree) {
         sb.append(" ").append(((IdentifierTree) t).name());
       } else if (t instanceof NativeTree) {
         sb.append(" ").append(((NativeTree) t).nativeKind());
+      } else if (t instanceof VariableDeclarationTree) {
+        sb.append(" ").append(((VariableDeclarationTree) t).identifier().name());
```

Comment:
I don't think it is necessary to append the identifier name here since it will be printed anyway when iterating over the declaration children